### PR TITLE
feature/104

### LINKED ISSUE #104: Airport Revisit: Flughafen München
See openscope/openscope#98

### PR DESCRIPTION
**Pull request by indianbhaji**
Munich Airport (IATA: MUC, ICAO: EDDM), German: Flughafen München, is a major international airport near Munich, the capital of Bavaria. It is the second busiest airport in Germany in terms of passenger traffic behind Frankfurt Airport, and the seventh busiest airport in Europe, handling 40,9 million passengers in 2015. It is the world's 15th busiest airport in terms of international passenger traffic, and was the 34th busiest airport worldwide in 2015. As of March 2015, the airport features flights to 228 destinations in 66 countries. Munich Airport serves as the secondary hub for Lufthansa including Lufthansa Regional and its Star Alliance partners besides Frankfurt.

The airport is located 28.5 km (17.7 mi) northeast of Munich near the old city of Freising and is named in memory of the former Bavarian Prime Minister Franz Josef Strauss. It features two passenger terminals with an additional midfield terminal and two runways as well as extensive cargo and maintenance facilities and is fully equipped to handle wide-body aircraft including the Airbus A380.

From [Wikipedia](https://en.wikipedia.org/wiki/Munich_Airport)

Also known as zlsa/atc#665
Part of zlsa/atc#650

Resolves #104.

@erikquinn When you review this, the ELMOX1C, NIRGO1C, VESUB1C, LULAR1C and all the -08 and -26 'STARs' are transitions. They've been placed for the future, and won't feature in spawnPatterns - I think these transitions would be nice for the users to use in order to guide aircraft to the runway and will be documented with #29.

### CODE REVIEW

File: assets/airports/eddm.json [hunk 1]
```diff
@@ -1,883 +1,2695 @@
...
-        "8L":  ["MNW", ["DM071", "K230"], ["DM093", "K250"], "DM092", "DM467"],
-        "8R":  ["MSW", ["DM080", "K230"], ["DM092", "K250"], "DM467"],
-        "26L": ["DM049", ["DM050", "K210"], ["MUN", "K250"], "LAKOL"],
-        "26R": ["DM067", ["DM050", "K210"], ["MUN", "K250"], "LAKOL"]
-      },
-      "body": ["ROTAX"],
-      "draw": [["DM093", "DM092"], ["DM092", "DM467", "ROTAX"], ["MUN", "LAKOL", "ROTAX"]]
+    "airspace": [
+        {
+        "floor": 0,
```
Comment (erikquinn) [inline assets/airports/eddm.json]: check indentation

File: assets/airports/eddm.json [hunk 2]
```diff
@@ -1,883 +1,2695 @@
...
+        "DM464":    ["N48d06m61.00",  "E011d53m11.00"],
+        "DM463":    ["N48d06m14.00",  "E011d47m04.00"],
+        "LULAR":    ["N48d05m20.00",  "E011d34m91.00"],
+        "VESUB":    ["N49d21m95.00",  "E012d45m53.00"],
+        "BIGNU":    ["N49d25m16.00",  "E012d59m85.00"],
+        "NIRGO":    ["N49d19m86.00",  "E013d07m93.00"],
+        "PIGAB":    ["N49d11m90.00",  "E010d08m60.00"],
+        "DKB"  :    ["N49d08m57.00",  "E010d14m30.00"],
+        "LEVBU":    ["N49d00m42.00",  "E010d27m40.00"],
+        "ELMOX":    ["N49d23m02.00",  "E009d49m48.00"]
```
Comment (erikquinn) [inline assets/airports/eddm.json]: Unless there's a significant advantage to keeping these grouped as they are, might as well sort them alphabetically (and check for duplicates!). :smile:

File: assets/airports/eddm.json [hunk 3]
```diff
@@ -1,883 +1,2695 @@
...
-        "26R": ["DM067", ["DM050", "K210"]]
-      },
-      "body": [["MUN", "K250"], "MANAL", "TULSI"],
-      "draw": [["DM071", "DM074", "DM082", "DM086", "MUN", "MANAL", "TULSI"]]
+    "runways":[
+        {
+            "name": ["26L", "08R"],
+            "name_offset": [[-1.5, -1.5], [1.5, -1.1]],
+            "end": [["N48.344783", "E11.804606", "1470ft"], ["N48.340656", "E11.750994", "1486ft"]],
+            "delay": [20, 30],
```
Comment (erikquinn) [inline assets/airports/eddm.json]: This parameter is not used; please remove.

File: assets/airports/eddm.json [hunk 4]
```diff
@@ -1,883 +1,2695 @@
...
+                "BIBAG": ["BIBAG"]
+            },
+            "draw": [
+                ["DM071", "RATGI"],
+                ["DM080", "RATGI", "BIBAG"],
+                ["DM049", "DM050"],
+                ["DM067", "DM050", "MUN", "MEBEK", "BIBAG"]
+            ]
+        },
+        "BIBAG2W": {
```
Comment (erikquinn) [inline assets/airports/eddm.json]: I see that you have `BIBAG2W` _and_ `BIBAG2S` from runway 26L. Are we able to remove one? Or do they actually use two different variants of the same arrival over the same fix going to the same runway?

File: assets/airports/eddm.json [hunk 5]
```diff
@@ -1,883 +1,2695 @@
...
+                "26L": ["DM452", "DM453", "DM454", ["DM455", "A80"], "DM456", "DM457", "DM458", "DM459"],
+                "26R": ["DM452", "DM453", "DM454", ["DM455", "A80"], "DM456", "DM457", "DM458", "DM459"]
+            },
+            "draw": [
+                ["OSDER", "DISUN", "BETOS"],
+                ["BETOS", "DM472", "DM473", "DM474", "DM475", "DM476", "DM457", "DM456", "DM455", "DM454", "DM453", "DM452", "DM451", "DM450"],
+                ["BETOS", "DM452", "DM453", "DM454", "DM455", "DM456", "DM457", "DM458", "DM459"]
+            ]
+        },
+        "ROKIL08": {
```
Comment (erikquinn) [inline assets/airports/eddm.json]: All STARs will need a `rwy` section, populated with (at least) empty or populated arrays for each runway for which the procedure is valid.

File: assets/airports/eddm.json [hunk 6]
```diff
@@ -1,883 +1,2695 @@
...
+            "name": "Betos Zero Eight",
+            "entryPoints": {
+                "BETOS": ["BETOS"]
+            },
+            "body": ["DM472", "DM473", "DM474", "DM475", "DM476", "DM457", "DM456", "DM455", "DM454", ["DM453", "A80"], "DM452", "DM451", "DM450"],
+            "draw": [
+                ["BETOS", "DM472", "DM473", "DM474", "DM475", "DM476", "DM457", "DM456", "DM455", "DM454", "DM453", "DM452", "DM451", "DM450"]
+            ]
+        },
+        "BETOS26": {
```
Comment (erikquinn) [inline assets/airports/eddm.json]: Are these `BETOS08` and `BETOS26` procedures applicable by making a `BETOS` arrival with suffixes `"08": "08", "26": "26"`?

We should do as much as we can to simplify these files, both to reduce their size, and to increase their readability.

File: assets/airports/eddm.json (hunk 4 above)
Comment (erikquinn) [inline assets/airports/eddm.json]: >@indianbhaji 
> The BIBAG2W at Munich is used if an aircraft cannot comply with the climb restriction

Is it safe to say they aren't used commonly? I suppose there's no sense removing them if they are actually occasionally used, but if they aren't (and few people will be familiar enough to know they exist in all likelihood), maybe we can just toss them. What do you think?

Comment (indianbhaji) [inline assets/airports/eddm.json]: @erikquinn toss them, I don't mind! haha

File: assets/airports/eddm.json [hunk 7]
```diff
@@ -1191,7 +1011,7 @@
             "altitude": null,
             "speed": null,
             "method": "random",
-            "rate": 1.3125,
+            "ra te": 1.3125,
```
Comment (erikquinn) [inline assets/airports/eddm.json]: accidental space

**Pull request merged**